test/core/rules/test_cases/README.md

```diff
@@ -0,0 +1,53 @@
+# Rule Tests
+
+All the individual rule unit tests are defined in yml files.
+
+Note that all the enumerated test names (test_1 etc.) were copied over from a previous format with an automated script. All new tests should be named descriptively with comments for more context if needed.
```

Comment:
We should also add an issue to add better test case names to these old tests as well. I can take care of that